kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -813,6 +813,9 @@ protected static List<String> getAuthenticatorCommandFromExecConfig(ExecConfig e
     }
     List<String> argv = new ArrayList<>(Utils.getCommandPlatformPrefix());
     command = getCommandWithFullyQualifiedPath(command, systemPathValue);
+    if (command.contains(" ")) {
```

Comment:
PodOperationsImpl has a shellQuote method that may be applicable here and to the arguments - since they are being joined via a space.  If they are all supposed to be literals, then using single quotes should be fine.  If interpolation is expected, then double quotes are required.

Reply:
I have updated but slightly change it with double quotes since single quotes seem not to be working as expected on windows.